jobs/config.json

```diff
@@ -7074,6 +7074,114 @@
       "sig-scalability"
     ]
   },
+  "ci-kubernetes-e2e-gke-sd-logging-gci-1-8": {
+    "args": [
+      "--check-leaked-resources",
+      "--deployment=gke",
+      "--extract=ci/latest-1.8",
```

Comment:
if you want to make the job flow with version, let's use `--extract=ci/k8s-stable1` and `--extract=ci/k8s-beta` here, per https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs

Reply:
Nice! Thanks, addressed it